test/e2e/runner/load.go

```diff
@@ -55,8 +56,9 @@ func Load(ctx context.Context, testnet *e2e.Testnet) error {
 		case <-chSuccess:
 			success++
 			timeout = stallTimeout
-		case <-chFailed:
+		case err := <-chFailed:
 			failed++
+			errorCounter[err.Error()]++
```

Comment:
Shouldn't we use the type here?

Reply:
You mean to use `error` as map key? That's not possible because `error` is not comparable.

Reply:
https://go.dev/play/p/RbGVvqwAAap ?

Reply:
anyway, it's not super important.

Reply:
yeah, I've tried that before, but for some reason, it didn't work because each error will be considered different, even when the corresponding strings are the same.